### PR TITLE
Update "Compare to Calibration" tutorial into "Identifying Hardware Changes" with Simultaneous Readout and additional context

### PR DESCRIPTION
**Pull request by augustehirth**
A large rework of the "Compare to Calibration" tutorial to emphasize the use of Parallel XEB and Simultaneous Readout (new) to identify if and how qubits have drifted since the previous maintenance calibration. Adds contextual information on what drift is, how it occurs, and what you can do about it once spotted. 

TODO/Things I don't have answers for yet: 
- Relative linking for API reference pages
- Need more links/references on qubit picking/mapping/routing. 
- An email address should be supplied for the quantum engine support team at the end of #Heatmap Observations
- Edit: Renaming the file without losing the previous contributors' commit history. 

More importantly, looking for any corrections, feedback or advice on improving this. Thanks!

### CODE REVIEW

**Review by MichaelBroughton — CHANGES_REQUESTED** [review]
Thanks so much for opening the PR. This is a good first pass on an update for this tutorial, there are a few outstanding changes that need to be made before we can get closer to merging. 

Also, if we are going to change the name of the tutorial, we will need to update https://github.com/quantumlib/Cirq/blob/master/docs/_book.yaml as well to update the name.

A lot of the reasoning behind some of the suggestions for changes can be found here: https://developers.google.com/style.

File: docs/tutorials/google/identifying_drift.ipynb [hunk 1]
```diff
@@ -0,0 +1,653 @@
...
+        "id": "HkVEdJnyGaax"
+      },
+      "source": [
+        "# Qubit Error Metrics\n",
+        "\n",
+        "This tutorial will focus on the following two metrics for our examples:\n",
+        "\n",
+        "*    Two Qubit Sqrt ISWAP XEB Pauli Error: For a particular pair of qubits, when we apply a $\\sqrt{i\\mathrm{SWAP}}$ gate, the Pauli error is the estimated probability with which the quantum state will depolarize as if a Pauli gate was applied to one or more of the qubits. We collect this measure with [Cross Entropy Benchmarking](../../qcvv/xeb_theory.ipynb) aka XEB because this is the method used during maintenance calibration\n",
+        "*    Parallel P11 Error: For each single qubit, we initialize the qubit to $|1⟩$ and measure the frequency with which the qubit's readout operation measures a $|1⟩$ state. The P11 error is the expected probability that the readout does not correctly measure $|1⟩$. We perform this experiment for all the qubits simultaneously, or in parallel. \n",
+        "\n",
```
[the remark this answers is not in the document]
Comment (augustehirth) [reply docs/tutorials/google/identifying_drift.ipynb:121]: Just double checking, you mean stick to 2nd person?

**Comment by augustehirth** [comment]
Thank you for all the detailed feedback. I'm working through the changes now.

File: docs/tutorials/google/identifying_drift.ipynb [hunk 2]
```diff
@@ -0,0 +1,653 @@
...
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 532
+        },
+        "id": "o4SMWcQcjxvg",
+        "outputId": "81b2dc67-7347-471c-c743-de870803da61"
+      },
+      "source": [
+        "sq_result = cirq.estimate_parallel_single_qubit_readout_errors(sampl
```
Comment (dstrain115) [inline docs/tutorials/google/identifying_drift.ipynb:631]: it's two-qubit to its two-qubit.

Comment (dstrain115) [inline docs/tutorials/google/identifying_drift.ipynb:629]: This analysis is quite confusing and hand-wavy.  Is this really our best practice?  Our advice is to stare at these heatmaps and try to manually glean some information out of them?  We are not going to tell them to refit bad two-qubit gates or to use any readout mitigation techniques?

Comment (dstrain115) [inline docs/tutorials/google/identifying_drift.ipynb:587]: Would it be worth plotting the difference between the two heatmaps on a third plot?

Comment (dstrain115) [inline docs/tutorials/google/identifying_drift.ipynb:648]: I think each of these tutorials actually use client-side calibration techniques and not the Calibration API.

Comment (augustehirth) [reply docs/tutorials/google/identifying_drift.ipynb:648]: This is an especially important point that I don't know how to immediately resolve. I attempted to use the shortest available way to collect the data, and focus the tutorial on comparison. This means using `cirq.experiments.xeb_sampling.sample_2q_xeb_circuits` and `cirq.estimate_parallel_single_qubit_readout_errors`, which, as far as I can tell, don't use the Calibration API under the hood. The alternative would be to set up and run a `cirq.google.CalibrationLayer` manually, or sending a large dummy circuit into `prepare_characterization_for_operations`. This is an option but doesn't seem to be presented the other tutorials. For now I'll fix the references to the Calibration API.

**Comment by augustehirth** [comment]
Version 3 has significant changes based on feedback, including removal of incorrect use of "drift", as well as addition/rework of "hardware changes" section.

**Comment by MichaelBroughton** [comment]
Any updates on the CLA status here @augustehirth ?

**Comment by augustehirth** [comment]
CLA status is moving forwards, but very slowly. Thanks for your patience.

**Comment by augustehirth** [comment]
@googlebot I signed it!

**Comment by augustehirth** [comment]
@MichaelBroughton CLA _should_ be resolved, is there anything else that needs to be changed here?

**Review by MichaelBroughton — APPROVED** [review]
Nope, LGTM.

**Pull request merged**